_docs_explore/64-request-items.md

```diff
@@ -1,4 +1,365 @@
+
+To request an accept/reject decision from a peer to a free text, the ConsentRequestItem is used.
+
+Please do not use the ConsentRequestItem to submit tons of text to the peer Identity. It is meant to be a short consent or summary the user agrees to. Please move longer text to external websites.
+The ConsentRequestItem is also not meant for contractual agreements.
+{: .notice--info}
+
+### Examples
+
+-   "I hereby confirm that I have read the privacy terms of this cloud service and agree to them."
+-   "The provided EULA has been read and agreed to."
+-   "Yes, I have backed up all of my data of this PC and you can wipe it."
+-   "I opt in to the newsletter."
+
+### Properties
+
+| Name      | Type                    | Description                                                                                                                                                                     |
+| --------- | ----------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `@type`   | `"ConsentRequestItem"`  | Specifies the type of the RequestItem for internal processing.                                                                                                                  |
+| `consent` | `string`                | The textual consent the user needs to give. This is different to the title and description of the RequestItem, as the consent would be the actual statement the user agrees to. |
+| `link`    | `string` \| `undefined` | A link to a website which contains more information, like the EULA or privacy terms.                                                                                            |
+
+### Response
+
+#### Item Properties
+
+-   To accept this RequestItem an [AcceptResponseItem]({% link _docs_explore/61-data-model.md %}#acceptresponseitem) will be transferred.
+-   To reject this RequestItem a [RejectResponseItem]({% link _docs_explore/61-data-model.md %}#rejectresponseitem) will be transferred.
+-   In case of an error an [ErrorResponseItem]({% link _docs_explore/61-data-model.md %}#errorresponseitem) will be transferred.
+
+#### Parameters
+
+-   To accept this RequestItem you can send `{ "accept": true }` as parameters.
+-   To reject this RequestItem you can send `{ "accept": false }` as parameters.
+
+## CreateAttributeRequestItem
+
+If you want to create Identity- or RelationshipAttributes for the peer, the CreateAttributeRequestItem can be used. Please have a look at the ProposeAttributeRequestItem if the peer should be able to overwrite the Attribute.
+
+To create a peer Attribute (owner = peer) with a fixed value defined by the sender, an Identity uses the CreateAttributeRequestItem. A fixed value in this case means, that the recipient is not allowed to change the value when accepting the request.
```

Comment:
To create a peer Attribute (owner = peer)

Es können auch eigene Relationship Attributes (=> owner = self) mit diesem Item verschickt werden.